caproto/threading/client.py

```diff
@@ -465,6 +466,9 @@ def get_cached_search_result(self, name, *,
 
             # Clean up expired result.
             self.search_results.pop(name, None)
+            pv_name_logger = logging.getLogger(f'caproto.bcast.search.{name}')
```

Comment:
We never need to log to _both_ `caproto.bcast.search.{name}` and `caproto.bcast.search`. Any messages logged to `caproto.bcast.search.{name}` will propagate to `caproto.bcast.search` as well.

I think it's open question whether we want to log this particular action. It may generate too much noise. We need a balance of giving enough detail to allow people to debug without overwhelming with detail. Let's circle back to this before we merge and decide whether it is useful or too much information; I'm not sure.